libdevice/CMakeLists.txt

```diff
@@ -0,0 +1,165 @@
+# Utility project providing various functionalities for SPIR-V devices
+# without native support of these functionalities.
+
+if (WIN32)
+  set(binary_dir "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}")
+else()
+  set(binary_dir "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
+endif()
```

Comment:
Is this var used? If so, where? And also it duplicates var on line 148

Reply:
${binary_dir} is used for the output files in all add_custom_command() calls below.
LIne 148 sets ${install_dest}, not ${binary_dir}.